src/http.ts

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+import express, { Request, Response } from 'express';
+import cors from 'cors';
+import { SwarmMCPServer } from './mcp-service';
+import { StreamableHTTPServerTransport } from '@modelcontextprotocol/sdk/server/streamableHttp.js';
+
+const port = process.env.PORT ? parseInt(process.env.PORT, 10) : 3000;
+const host = process.env.HOST || '0.0.0.0';
+
+const app = express();
+
+// Enable CORS for all routes
+app.use(cors());
+app.use(express.json());
+
+// Handle all MCP requests on the /mcp endpoint
+app.all('/mcp', async (req: Request, res: Response) => {
```

Comment:
I cannot try it out with Windsurf as that [supports only SSE](https://docs.windsurf.com/windsurf/cascade/mcp#mcp-config-json).